docs/docs/building-classic-assistants/testing-your-assistant.mdx

```diff
@@ -0,0 +1,554 @@
+---
```

Comment:
In this file, I kept everything but the section on end-to-end testing, and linked to the page in `production/` folder